civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1166,3 +1189,23 @@ function civihr_employee_portal_views_pre_view(&$view, &$display_id, &$args) {
     );
   }
 }
+
+/**
+ * @param mixed $part The $query->where part
```

Comment:
According to the Drupal code standards, the @param description should be in a new line https://www.drupal.org/node/1354#param